org.moreunit.swtbot.test/src/org/moreunit/JavaProjectSWTBotTestHelper.java

```diff
@@ -130,11 +129,11 @@ private static void closeWelcomeViewIfActive()
     protected void openResource(String resourceName)
     {
         KeyboardFactory.getAWTKeyboard().pressShortcut(getCmdOrStrgKeyForShortcutsDependentOnPlattform() | SWT.SHIFT, 'r');
-        SWTBotShell openTypeShell = bot.shell("Open Resource");
-        assertThat(openTypeShell).isNotNull();
-        openTypeShell.activate();
+        //WORKAROUND due to an SWTBot bug when shell title is updated afterwards
+        bot.shells();
+        bot.waitUntil(Conditions.shellIsActive("Open Resource"));
```

Comment:
The 'workaround' code should be extracted in to a method, then it could be reused in `RenameMethodTest` as well. Plus the workaround could be remove in a single place when not necessary any more.